spec/core/matchers/toHaveSpyInteractions.js

```diff
@@ -0,0 +1,57 @@
+describe('toHaveSpyInteractions', () => {
+  let spyObj;
+  beforeEach(() => {
```

Comment:
I'd rather see each spec contain just the setup it needs, inline. De-duplicating test setup like this introduces coupling between tests and reduces locality. Sometimes that's worthwhile, but IMO not in this case where we're looking at one line of setup in most specs.

Reply:
I was following DRY principle since the same setup is required for all tests except for one, but I do not have a strong opinion here. Hence, moved the setup to each test call.